File: HelpMEApp/HelpMEWebPart/src/app/model/UserWhoAttending.ts
/**
 * Created by Hrvoje on 02.06.2016..
 */

export class UserWhoAttending{

  public Id: number;
  public Name: string;
  public Surname: string;
  public UserPhoto: string;


    constructor(Id:number, Name:string, Surname:string, UserPhoto:string) {
      this.Id = Id;
      this.Name = Name;
      this.Surname = Surname;
      this.UserPhoto = UserPhoto;
    }
}

File: HelpMEApp/HelpMEServices/src/HelpMEServices/Models/UserWhoAttending.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace HelpMEServices.Models
{
    public class UserWhoAttending
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string UserPhoto { get; set; }

        [ForeignKey("AppEventId")]
        public int AppEventId { get; set; }
        public AppEvent AppEvent { get; set; }
        
    }
}
